lib/containers/runtime.pm

```diff
@@ -32,22 +34,71 @@ sub _rt_script_output {
     return script_output($self->runtime . " " . $cmd, @args);
 }
 
-=head2 build
+=head2 create_container($image, $name, [%args])
+
+Creates a container.
+C<image> the name of the image we create the container from.
+C<name> is the given name of the created container.
+C<args> can be anything you want to run in the container, similar to C<run>. for 
+instance: C<docker create -it tumbleweed bash>
+
+=cut
+sub create_container {
+    my ($self, $image, $name, $args) = @_;
+    die 'wrong number of arguments' if @_ < 3;
+    $self->_rt_assert_script_run("container create --name $name $image $args", 300);
+    record_info "$name container created", "";
+}
+
+=head2 start_container($image_name)
+
+Starts container named C<image_name>.
+
+=cut
+sub start_container {
+    my ($self, $image_name) = @_;
+    die 'wrong number of arguments' if @_ < 2;
+    $self->_rt_assert_script_run("container start $image_name");
+    record_info "$image_name container started", "";
+}
+
+=head2 halt_container($container_name)
```

Comment:
I don't think that `halt_container()` means `Blocks a container until exits.`. I'd suggest something like `wait_for_container()` instead.

Reply:
matter of taste here i guess. by definition `halt`=="a suspension of movement or activity, typically a temporary one." 
i wanted to avoid to name it `wait_container` and for the same reason i dont really like `wait_for_container` as the name describe a behavior, and the those ones could be anything. 

Also the [documentation](https://docs.docker.com/engine/reference/commandline/wait/) has similar description
`Block until one or more containers stop, then print their exit codes`. that isnt big different than `Blocks a container until exits` IMO

Reply:
OK.